boards/arm/96b_meerkat96/doc/index.rst

```diff
@@ -0,0 +1,330 @@
+  - 1x Camera Parallel Interface
+  - 1x Analog Audio Line in (Stereo)
+  - 1x Analog Audio Mic in (Mono)
+  - 1x Analog Audio Headphone out (Stereo)
+
+- Connectivity
+
+  - USB 2.0 OTG (High Speed)
+  - USB 2.0 host (High Speed)
+  - 10/100 Mbit/s Ethernet PHY
+  - 4x I2C
+  - 4x SPI
+  - 7x UART
+  - 1x IrDA
+  - 20x PWM
+  - Up to 125 GPIO
+  - 4x Analog Input (12 Bit)
+  - 2x SDIO/SD/MMC (8 Bit)
+  - 2x CAN
+
+More information about the i.MX7 SoC can be found here:
+
+- `i.MX 7 Series Website`_
+- `i.MX 7 Dual Datasheet`_
+- `i.MX 7 Dual Reference Manual`_
+
+Supported Features
+==================
+
+The Zephyr 96b_meerkat96 board configuration supports the following hardware
+features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+
+The default configuration can be found in the defconfig file:
+
+	``boards/arm/96b_meerkat96/96b_meerkat96_defconfig``
+
+Other hardware features are not currently supported by the port.
+
+Connections and IOs
+===================
+
+96Boards Meerkat96 board was tested with the following pinmux controller
+configuration.
+
++---------------+-----------------+---------------------------+
+| Board Name    | SoC Name        | Usage                     |
++===============+=================+===========================+
+| UART_1 RXD    | UART1_TXD       | UART Console              |
++---------------+-----------------+---------------------------+
+| UART_1 TXD    | UART1_RXD       | UART Console              |
++---------------+-----------------+---------------------------+
+| LED_R1        | GPIO1_IO04      | LED0                      |
++---------------+-----------------+---------------------------+
+| LED_R2        | GPIO1_IO05      | LED1                      |
++---------------+-----------------+---------------------------+
+| LED_R3        | GPIO1_IO06      | LED2                      |
++---------------+-----------------+---------------------------+
+| LED_R4        | GPIO1_IO07      | LED3                      |
++---------------+-----------------+---------------------------+
+
+System Clock
+============
+
+The M4 Core is configured to run at a 200 MHz clock speed.
+
+Serial Port
+===========
+
+The iMX7D SoC has seven UARTs. The number 2 is configured for the console and
```

Comment:
Instead of ``The number 2 is configured...`` would this be clearer as ``UART_1 is configured...``